pkg/testgridanalysis/testidentification/test_identification.go

```diff
@@ -74,9 +74,9 @@ var curatedTestSubstrings = map[string][]string{
 }
 
 var (
-	cvoAcknowledgesUpgrade = "[sig-cluster-lifecycle] Cluster version operator acknowledges upgrade"
-	operatorsUpgraded      = "[sig-cluster-lifecycle] Cluster completes upgrade"
-	machineConfigsUpgraded = "[sig-mco] Machine config pools complete upgrade"
+	cvoAcknowledgesUpgrade = "Cluster upgrade.[sig-cluster-lifecycle] Cluster version operator acknowledges upgrade"
+	operatorsUpgraded      = "Cluster upgrade.[sig-cluster-lifecycle] Cluster completes upgrade"
+	machineConfigsUpgraded = "Cluster upgrade.[sig-mco] Machine config pools complete upgrade"
```

Comment:
this is going to cause some trouble w/ doing reporting on historical files which now won't match this.

Can you instead make the consumers of these constants do a regex or substring comparison instead of an exact equals check?  If you make it a regex we can include a $ terminator to ensure it's still a pretty precise match that's unlikely to get false positives against something else.

Reply:
:+1: